client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -119,6 +142,11 @@ export const useJsonForms = (
   const { success, error: errorNotification } = useNotification();
   const navigate = useNavigate();
 
+  // fetch document (only if there is as document name)
+  const { data: databaseResponse, isLoading } = useDocument.document.get(
+    documentName ?? '',
+    documentName !== undefined
```

Comment:
```suggestion ts
    !!documentName
```

Maybe nicer?

Reply:
actually prefer the first one, IMHO easier to read and !! does some falsy conversions as well which might not be what you wanted to check... e.g. if you have a `const var: string | undefined` and `var` is an empty string `!!var` is not equivalent to `var !== undefined`.

Reply:
We seem to `!!` a lot more then !== undefined (if you do global search, 7 vs 127 results). 
`// fetch document (only if there is as document name)` is it possible to have document with empty name ?

Reply:
my point is with `!== undefined` you don't need to think about these details, e.g. if string can be empty. I can use !! here but its a different check.